.circleci/config.yml

```diff
@@ -788,6 +788,7 @@ workflows:
           name: deploy bitcoin develop (shapeshift)
           organization: SHAPESHIFT
           pulumi-command: up -f --yes
+          daemon-storage-size: 650Gi
```

Comment:
Won't this be written over by the `<< : *bitcoin-dev` since the value is defined there as well?